DigitalLibraryofTennessee/dltn_ingestion3_config_generator
# Add five sets from Montgomery County Archives for March ingest.

**GitHub Issue**: [#277](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/277)

## What does this Pull Request do?

Adds five sets for Montgomery County Archives to config and sets list.

## How should this be tested?

Check to make sure the set names match those in Repox. Run check and harvest on the added sets.
